MapboxNavigation/RouteMapViewController.swift

```diff
@@ -311,6 +315,33 @@ class RouteMapViewController: UIViewController {
         statusView.show(title, showSpinner: true)
     }
     
+    @objc func progressDidChange(notification: NSNotification) {
+        guard showMaximumSpeedLimitSign else {
+            navigationView.speedLimitSign.isHidden = true
```

Comment:
Move this line above the `guard` statement and change it so that it can show the sign if necessary:

```swift
navigationView.speedLimitSign.isHidden = !showsMaximumSpeedLimitSign || !hasSpeedLimit
```

Then remove the last line of this method. If it’s a problem that we’re showing the sign before performing layout, then stick the line in a `defer` block.